src/create/create.js

```diff
@@ -39,6 +39,28 @@ module.exports = function create(namespace, statics, proto) {
         return origin;
     }
 
+    function extendMixins(origin,mixins) {
+        for (var i = 0; i < mixins.length; i++) {
+            var mixinsToMerge = {};
+            // if the mixin is a jig
+            if(typeof mixins[i] === 'function'){
+                var jig = new mixins[i];
```

Comment:
You do not need to create a new instance here. Just copy/extend the static functions and the prototype function.